tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/functions/FunctionSignatureInBalaTest.java

```diff
@@ -155,6 +156,15 @@ public void testInvokeFunctionWithRequiredArgsOnly() {
         Assert.assertEquals(returns[5].stringValue(), "[]");
     }
 
+    @Test
+    public void testInvokeFunctionWithAnyFunctionTypeParam() {
+        BValue[] returns = BRunUtil.invoke(result, "testAnyFunction");
```

Comment:
We can write this test in Ballerina itself, right? Let's also add a negative bala test. We can do these in a separate PR.

Reply:
Create a separate [issue](https://github.com/ballerina-platform/ballerina-lang/issues/29112) to track this.